Assets/SEE/Controls/DesktopPlayerMovement.cs

```diff
@@ -22,11 +23,34 @@ private struct CameraState
 
         private CameraState cameraState;
 
+        /// <summary>
+        /// Unity component that moves the player constrained by collisions.
+        /// It moves the player with its own <see cref="CharacterController.Move(Vector3)"/> method.
+        /// </summary>
+        private CharacterController controller;
+
         [Tooltip("The code city which the player is focusing on.")]
         public GO.Plane FocusedObject;
 
         private void Start()
         {
+            GameObject head = GameObject.Find("/Local Player 1/Root/Global/Position/Hips/LowerBack/Spine/Spine1/Neck/Head/HeadAdjust");
```

Comment:
Note that `GameObject.Find` traverses all game objects of a scene and is, hence, a quite expensive operation. This may not be a problem here because `Start()` is called only once. Anyhow, you should not rely on the name of the player being `Local Player 1`.

There is a better way to find a child: [Transform.Find](http://unity3d.com/support/documentation/ScriptReference/Transform.Find.html). 

Moreover, be prepared that the result could be null.

I added a local method `HeadRadius()` that considers all that:

```cs
            // Returns the radius of the player's head.
            float HeadRadius()
            {
                const string headName = "Root/Global/Position/Hips/LowerBack/Spine/Spine1/Neck/Head/HeadAdjust";
                Transform head = transform.Find(headName);
                if (head == null)
                {
                    Debug.LogError($"Player {gameObject.name} does not have a child {headName}.\n");
                    return 1.0f;
                }
                // We want to fit the head completely into the CharacterController collider, so we use the maximum value
                // to calculate the radius
                return Mathf.Max(head.transform.localScale.x, head.transform.localScale.y, head.transform.localScale.z) / 2;
            }
```

Reply:
I fixed it.